packages/suite/src/views/wallet/transactions/TradeBox/TradeBoxMenu.tsx

```diff
@@ -38,37 +30,35 @@ interface TradeBoxMenuProps {
 export const TradeBoxMenu = ({ account }: TradeBoxMenuProps) => {
     const dispatch = useDispatch();
 
-    const menuItems: (OverflowItem & {
+    const menuItems: {
         route: Route['name'];
-        type: 'exchange' | 'buy' | 'sell' | 'spend';
+        type: 'exchange' | 'buy' | 'sell';
         title: ReactNode;
-    })[] = [
+        hideable?: boolean;
```

Comment:
nitpick: call it `isHideable` please for consistency (we tend to use "is" prefix for booleans if it makes sense)